.circleci/config.yml

```diff
@@ -60,7 +61,7 @@ jobs:
 
   sam_build:
     docker:
-      - image: amazon/aws-sam-cli-build-image-python3.10:latest
+      - image: amazon/aws-sam-cli-build-image-python3.8:latest
```

Comment:
Flagging this change in case it might cause deployment issues. Increasing to 3.10 caused a build error.

Reply:
Have a read of this fun issue https://github.com/aws/aws-lambda-base-images/issues/31

Including my favourite comment:

> If your release cycle of Lambda containers with new Python versions takes longer than that of completely new Python versions, I'd say, there's something wrong with your release process.

tl;dr: AWS Lambda doesn't support 2.10 yet :(

Reply:
Oh yes, I remember you mentioning this in Slack. So I suppose we hold off on upgrading?

Reply:
I've [added comment for aws-lambda-base-images](https://github.com/DemocracyClub/Website/pull/628/commits/def0af5490e717e3dc14135e0c1f385e932b6f85)